loan-payment-skip/LETTERSPECS/BANNO.LOANPAYMENT.SKIP.CFG

```diff
@@ -207,7 +209,17 @@ DQE:0
 * should not be changed without first carefully evaluating the potential
 * repercussions.
 ************************************************************************
-*
+* PROGRAM INSTALLATION DATE
+*  Line must begin with "PID:"
+*
+*  Enter the date the program was activated in the production system.
+*  Additional program data will be accessible for 90 days after the
+*  installation date to assist with issues tracking.
+*  VALID VALUES: Any valid date on or prior to the system date. in the
+*                following format: mm/dd/yyyy.
+*  DEFAULT: In absence of a date, 90 days from the program's most recent
+*           modification date will be used.
+PID:
 * PROGRAM TIME PERIOD
 * The calender date range during which this skip program is available to

```

Comment:
@tkainz Code Looks Good!
I would just change the spelling of "calender" in the CFG on line 224 where is says "* The calender date range during"